### ISSUE TITLE
Need to check $url length before generate the iframe ?

### ISSUE THREAD

**Issue by COLINJP-dev** [wrav/oembed#86]
Since upgrading to the latest 1.3.11 version.

I think you need to check the $url length because we have a with $url with value "/?" that cause a iframe of frontend homepage in preview if the field is empty.

**Comment by reganlawton** [comment]
Can you explain this able more?

**Comment by COLINJP-dev** [comment]
Hi,

This code added in 1.3.9
```
if (!preg_match('/\?(.*)$/i', $src)) {
                    $src .= "?";
                }```

Cause the url never empty and the preview Will show you your frontend

**Comment by COLINJP-dev** [comment]
Fixed on 1.3.12
